packages/manager/modules/carrier-sip/src/cdr/cdr.html

```diff
@@ -0,0 +1,43 @@
+<section>
+    <tuc-toast-message></tuc-toast-message>
+    <h3 data-translate="carrier_sip_cdr_title"></h3>
+    <p data-translate="carrier_sip_cdr_information"></p>
+    <form
+        novalidate
+        data-ng-submit="$ctrl.getCallDetailRecords()">
+        <oui-spinner data-ng-if="$ctrl.isExporting"></oui-spinner>
```

Comment:
This isn't handled by the routing ?

Reply:
It's not handled by the routing. In fact we want to let the ability to the user to get their call details records from multiple `month` by keeping it in the same state.

As soon as the polling is finished then we remove the `<oui-spinner>` and we display a success message. It means that now, the user is now able to perform a new request for another month/year for instance.

Is that make sense for you?

Reply:
Yup, works for me :)